docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3"
+
+services:
+  parrot:
+    container_name: parrot
+    build: .
+    # image: ghcr.io/aquelemiguel/parrot
+    env_file: .env
+    restart: unless-stopped
```

Comment:
I'd delete this whole file. We do not need this.

Reply:
Not sure I agree--nowadays, it's commonplace to include a compose file for single image projects. Running `docker-compose up` is pretty different from copying a really big `docker run` command from the documentation. It will also allow us to specifiy standard volume paths when implementing serialization for guild settings.

Reply:
Im with @afonsojramos. And even if it is handy in the future, we will cross that bridge when we get there. For the purpose of this PR, keep it to copying the cargo lock file